### PR TITLE
phoenix new and create objective table

### PR DESCRIPTION
**Pull request by kumabook**
Revise project with phoenix instead of rails

### CODE REVIEW

**Comment by kumabook** [comment]
@pankona @kasaharu I restart the project with elixir and phoenix.
if you are interested, please review it

**Comment by pankona** [comment]
@kumabook
i'm so happy to hear streakable's reincarnation!
plz let me contribute. i'll review.

**Comment by pankona** [comment]
@kumabook

revewed. here's my questions:

* it seems brunch will not be used since webpack is placed o instead. is it possible to remove files they are related to brunch?
* objective model has a field "frequncy" as integer. what kind of value will be stored on here? i'd like to know an example.

thanks.

**Comment by kumabook** [comment]
@pankona 
> it seems brunch will not be used since webpack is placed o instead. is it possible to remove files they are related to brunch?

removed 	brunch config file at 5fc11f9

> objective model has a field "frequncy" as integer. what kind of value will be stored on here? i'd like to know an example.

frequency means `how often a user should or want to contribute to specified objective`.
N (day / contribute)
For example,  every day ... 1, once a week 7.

**Comment by pankona** [comment]
@kumabook 

understood. thank you for explanation.
now I have no concern on this pull request !

**Comment by kasaharu** [comment]
@kumabook 
Sorry for my late reply.
I'll review tonight.

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
 # streakable
+
+## What's is this?
```
Comment (kasaharu) [inline README.md]: `What's is this` -> `What's this`

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+Streakable is the web service that helps you to keep tryin' like github streak.
+You can record your contributions for your target,
+and streakable visualizes your contributions.
+
+
+## Development
+
+1. Install [asdf][] and run `asdf install`
+2. Install posgresql and create postgres(pass: postgres) user
```
Comment (kasaharu) [inline README.md]: And give `CREATEDB` role?

File: .gitignore [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
+/_build
+/db
+/deps
+/*.ez
+
+# Generated on crash by the VM
+erl_crash.dump
+
+# Static artifacts
+/node_modules
```
Comment (kasaharu) [inline .gitignore]: Want to add `npm-debug.log` to this file.

File: README.md (hunk 1 above)
Comment (kumabook) [inline README.md]: oops! fixed at e4b66727bcb

File: README.md (hunk 2 above)
Comment (kumabook) [inline README.md]: OK. Add `createdb` command example.  0216a231c7c8e

File: .gitignore (hunk 3 above)
Comment (kumabook) [inline .gitignore]: OK added at bb506403a07b7d

**Comment by kasaharu** [comment]
@kumabook 
Thanks!
Looks good to me.

**Comment by kumabook** [comment]
@pankona @kasaharu 

Thanks a lot!

Anyone put the image, so I did

[LGTM](https://lgtm.in/i/2leFe8mpq)

[:+1:](https://lgtm.in/u/2leFe8mpq) [:-1:](https://lgtm.in/r/2leFe8mpq)

**Pull request merged**